code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -425,6 +426,7 @@
 	movement_target = null
 	icon_state = icon_dead
 	density = FALSE
+	stasis = TRUE
```

Comment:
Why are you using stasis for checks, rather than stat (LIVING, UNCONSCIOUS, DEAD)?

Reply:
Just because stasis check was already used there. Or it meant to be used, but "stasis" was never triggered, that caused Bones to speak after death, probably never stopped to process other stuff too.
It's also used in mob spawners code somewhere, so replacing stasis check with LIVING/UNCONSCIOUS/DEAD likely to break something.